docs/plugins/plugin-development-guide/naming.rst

```diff
@@ -0,0 +1,59 @@
+Naming changes
+--------------
+
+``PluginSkeleton`` provides some default classes and configurations. However, they must have some default values and names that should be changed
+to reflect your plugin functionality. Basing on the vendor and plugin names established above, these are the changes that should be made:
+
+* In ``composer.json``:
+
+    * ``sylius/plugin-skeleton`` -> ``iron-man/sylius-product-on-demand-plugin``
+
+    * ``Acme example plugin for Sylius.`` -> ``Plugin allowing to mark product variants as available on demand in Sylius.`` (or sth similar)
+
+    * ``Acme\\SyliusExamplePlugin\\`` -> ``IronMan\\SyliusProductOnDemandPlugin\\`` (the same changes should be done in namespaces in ``src/`` directory
+
+    * ``Tests\\Acme\\SyliusExamplePlugin\\`` -> ``Tests\\IronMan\\SyliusProductOnDemandPlugin\\`` (the same changes should be done in namespaces in ``tests/`` directory
+
+* ``AcmeSyliusExamplePlugin`` should be renamed to ``IronManSyliusProductOnDemandPlugin``
+
+* ``AcmeSyliusExampleExtension`` should be renamed to ``IronManSyliusProductOnDemandExtension``
+
+* In ``src/DependencyInjection/Configuration.php``:
+
+    * ``acme_sylius_example_plugin`` -> ``iron_man_sylius_product_on_demand_plugin``
+
+* In ``tests/Application/app/AppKernel.php``:
+
+    * ``\Acme\SyliusExamplePlugin\AcmeSyliusExamplePlugin()`` -> ``\IronMan\SyliusProductOnDemandPlugin\SyliusProductOnDemandPlugin()``
+
+* In ``phpspec.yml.dist`` (if you want to use PHPSpec in your plugin):
+
+    * ``Acme\SyliusExamplePlugin`` -> ``IronMan\SyliusProductOnDemandPlugin``
+
+That's it! All other files are just a boilerplate to show you what can be done in the Sylius plugin. They can be deleted with no harm.
```

Comment:
"They can be deleted with no harm." -> "They can be deleted with no harm:" in order to indicate, that the list below is directly connected with this sentence?